lib/js/dom.nim

```diff
@@ -1267,6 +1267,7 @@ proc inViewport*(el: Node): bool =
            rect.right <= clientWidth().float
 
 proc scrollTop*(e: Node): int {.importcpp: "#.scrollTop", nodecl.}
+proc scrollLeft*(e: Node): int {.importcpp: "#.scrollTop", nodecl.}
```

Comment:
<img width="259" alt="image" src="https://user-images.githubusercontent.com/68273/52726863-0228e800-2f82-11e9-9872-e4471ca2bfcd.png">

Careful with that `.importcpp` directive here.

Reply:
sorry, I checked but one stage change was still missed, it uses the right directive now.